integrated_channels/api/v1/logs/views.py

```diff
@@ -67,15 +119,43 @@ def get_serializer_class(self):
             raise exceptions.ParseError("Invalid channel code.")
         return serializer
 
+    def validate(self):
+        integrated_channel_code = self.kwargs.get('integrated_channel_code', None)
+        ThisLearnerClass = LearnerDataTransmissionAudit.get_completion_class_by_channel_code(integrated_channel_code)
+        if ThisLearnerClass is None:
+            raise InvalidChannelCodeException
+
+        if sort_by_param := self.request.GET.get('sort_by'):
+            if sort_by_param.replace("-", "") not in ["sync_status", "sync_last_attempted_at"]:
+                if sort_by_param.replace("-", "") not in [field.name for field in ThisLearnerClass._meta.fields]:
+                    raise InvalidSortByParamException
```

Comment:
Almost identical code to line 73 above, looks like we can wrap into helper method.